.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+*            @zepatrik
+
+/docs/docs/  @ory/documenters
```

Comment:
I think we could even give `/docs` as they could merge also changes to the template plus versioned docs :)

Reply:
ah right forgot versioned docs